### PR TITLE
Fix travis build

### LINKED ISSUE #22: Fix travis build
Travis is now configured at travis-ci.org/edelight/chef-solo-search, but the build fails with:

```
$ ruby -Ilibraries tests/test_search.rb -v
tests/test_search.rb:21:in `require': no such file to load -- chef (LoadError)
    from tests/test_search.rb:21

The command "ruby -Ilibraries tests/test_search.rb -v" exited with 1.
```

### PR DESCRIPTION
**Pull request by tjwallace**
Fixes #22

### CODE REVIEW

File: tests/test_search.rb [hunk 1]
```diff
@@ -192,7 +192,9 @@ def test_nested_fieldnames
 module SearchNodeTests
   def test_list_nodes
     nodes = search(:node)
-    assert_equal Chef::Node, nodes.first.class
+    assert_equal Chef::Node, nodes.find{ |n| n["hostname"] == "alpha.example.com" }.class
+    assert_equal Chef::Node, nodes.find{ |n| n["hostname"] == "beta.example.com" }.class
+    assert_equal Hash, nodes.find{ |n| n["hostname"] == "wjc.example.com" }.class
```
Comment (tjwallace) [inline tests/test_search.rb:198]: The not prettiest code ... I wonder if there is a better way

**Comment by tjwallace** [comment]
Strangely the build wasn't failing on my local machine.

**Comment by tjwallace** [comment]
Hmm 1.9.3 and 1.9.2 passed, but the [1.8.7 build](https://travis-ci.org/edelight/chef-solo-search/jobs/5347554) failed because it couldn't find the chef gem, even though you can see it being installed in the logs.

**Comment by tjwallace** [comment]
Fixed ... in 1.8.7 you have to explicitly require `rubygems`

**Comment by tobami** [comment]
Ahh, I see the "good to merge" now. Awesome, thanks!

**Pull request merged**